src/plugins/custom_import_map/public/components/empty_prompt.tsx

```diff
@@ -16,6 +16,7 @@ export interface EmptyPromptProps {
 const EmptyPrompt = (props: EmptyPromptProps) => {
```

Comment:
could combine L16 and L31 to `export default const EmptyPrompt = (props: EmptyPromptProps) => {`

Reply:
getting this error - ` Only expressions, functions or classes are allowed as the default export. `